models/project-extension/src/generator.ts

```diff
@@ -0,0 +1,39 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { CustomApplicationInstallation } from '@commercetools-test-data/custom-application';
+import { CustomViewInstallation } from '@commercetools-test-data/custom-view';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+
+import * as CategoryRecommendationSettings from './category-recommendation-settings';
+import * as ImageRegex from './image-regex';
+import * as SampleDataImport from './sample-data-import';
+
+import type { TProjectExtension } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TProjectExtension>({
+  fields: {
+    createdAt: fake(getOlderDate),
+    updatedAt: fake(getNewerDate),
+    id: fake((f) => f.string.uuid()),
+    projectKey: fake((f) => f.lorem.slug()),
+    imageRegex: fake(() => ImageRegex.random()),
+    applications: [], // Note: This is deprecated
```

Comment:
Let's omit it

Reply:
I left it to avoid type errors or using ts-ignore directive